src/components/ListGoals.js

```diff
@@ -3,14 +3,12 @@ import {Link} from "react-router-dom";
 import {Container} from "../styles/ListItem";
 import RepoListItem from "../components/RepoListItem";
 import Card from "../components/Card";
-import AddRepoForm from "../components/AddRepoForm";
 import List from "../styles/List";
 
 function ListGoals({goals, goalsId}) {
   return (
     <Container>
       <Card fitted>
```

Comment:
Remove this component here and move to the outer `<Goals />` component.

It currently causing a visual bug with the input padding:

<img width="420" alt="Screenshot 2019-11-11 06 09 03" src="https://user-images.githubusercontent.com/5713670/68593365-21d63980-044a-11ea-946e-84ae8d23c290.png">
 
Should be:

<img width="282" alt="Screenshot 2019-11-11 06 09 20" src="https://user-images.githubusercontent.com/5713670/68593373-2864b100-044a-11ea-9f66-87dee0e91b5a.png">